commands.go

```diff
@@ -469,8 +469,16 @@ func deleteRecord(name string, match string, rtype string, wait bool, identifier
 
 	match = qualifyName(match, *zone.Name)
 	changes := []*route53.Change{}
+	text := ""
 	for _, rrset := range rrsets {
```

Comment:
There's actually a helper 'unescaper' for doing these escape/unescaping - see unescaper.Replace is used in bind.go - this would be a better way.